contrib/builder/deb/arm/debian-jessie/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM armhf/debian:jessie
```

Comment:
@tianon This folder deb/arm/ is named by the values from Go / DOCKER_ENGINE_GOARCH. And there it is called "arm" for 32bit and "aarch64" for 64bit (AFAIK).
Introducing "armhf" here we would need another mapping / variable for that.